lib/streamlit/commands/page_config.py

```diff
@@ -295,6 +272,8 @@ def set_menu_items_proto(lowercase_menu_items, menu_items_proto) -> None:
     if ABOUT_KEY in lowercase_menu_items:
         if lowercase_menu_items[ABOUT_KEY] is not None:
             menu_items_proto.about_section_md = dedent(lowercase_menu_items[ABOUT_KEY])
+        else:
```

Comment:
Needed to handle case where `"About"` markdown intially set and then updated to `None`, for consistent behavior with other `menu_items` fields.

Reply:
nit: it would be good to add that as a comment in the code as well